### PR TITLE
Fix/xsimd scalar missing functions

### PR DESCRIPTION
**Pull request by serge-sans-paille**

### CODE REVIEW

**Comment by serge-sans-paille** [comment]
These ones are need for pythran.

File: include/xsimd/math/xsimd_scalar.hpp [hunk 1]
```diff
@@ -48,6 +49,7 @@ namespace xsimd
     using std::log1p;
     using std::log2;
     using std::log;
+    using std::modf;
```
Comment (wolfv) [inline include/xsimd/math/xsimd_scalar.hpp]: should we also add a test for modf? is it in xsimd?

Comment (serge-sans-paille) [reply include/xsimd/math/xsimd_scalar.hpp]: modf is not in the vector version of xsimd, so I did'nt want to add test for only the scalar mode. take that as an incentive to implement the vector version :-)

Comment (wolfv) [reply include/xsimd/math/xsimd_scalar.hpp]: :)

**Pull request merged**